applications/machine_learning/src/modules/ml_runner.c

```diff
@@ -19,26 +20,31 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_ML_APP_ML_RUNNER_LOG_LEVEL);
 #define SHIFT_WINDOWS		CONFIG_ML_APP_ML_RUNNER_WINDOW_SHIFT
 #define SHIFT_FRAMES		CONFIG_ML_APP_ML_RUNNER_FRAME_SHIFT
 
+#define ML_STATE_CONTROL	IS_ENABLED(CONFIG_ML_APP_ML_STATE_EVENTS)
```

Comment:
This name is very confusing as it blends with names in this new enum. See line 198 to know what I mean.

Reply:
You are right. Changed.